Project/Source/Modules/ModuleWindow.cpp

```diff
@@ -129,6 +139,44 @@ void ModuleWindow::SetTitle(const char* title) {
 	SDL_SetWindowTitle(window, title);
 }
 
+void ModuleWindow::SetCursor(bool isPlaying) {
+	if (isPlaying) {
+		// Load resource
+		Scene* scene = App->scene->scene;
+		ResourceTexture* cursorResourceTexture = App->resources->GetResource<ResourceTexture>(scene->GetCursor());
+		
+		// From glTexture to SDL_Surface
+		int w = 0;
+		int h = 0;
+		glGetTextureLevelParameteriv(cursorResourceTexture->glTexture, 0, GL_TEXTURE_WIDTH, &w);
+		glGetTextureLevelParameteriv(cursorResourceTexture->glTexture, 0, GL_TEXTURE_HEIGHT, &h);
```

Comment:
cursorResourceTexture can be nullptr if no cursor is set for the scene.

Engine crashes at glGetTextureLevelParameteriv() in that case.